lib/grape/middleware/error.rb

```diff
@@ -122,16 +85,51 @@ def format_message(message, backtrace, original_exception = nil)
 
       private
 
-      def rescue_all?
-        options[:rescue_all]
+      DEFAULT_RESCUE_HANDLER = :handle_error
```

Comment:
Seems pretty but maybe unnecessary? I mean `:handle_error` is already a constant, since this isn't going to change I say copy-paste it everywhere.

Reply:
... or maybe even rename it to `:default_rescue_handler`?